packages/core/chrome/core-chrome-browser/src/project_navigation.ts

```diff
@@ -58,11 +59,17 @@ export interface ChromeProjectNavigationNode {
   /** Optional children of the navigation node */
   children?: ChromeProjectNavigationNode[];
   /**
-   * Temporarilly we allow href to be passed.
-   * Once all the deeplinks will be exposed in packages we will not allow href anymore
-   * and force deeplink id to be passed
+   * href for absolute links only. Internal links should use "link".
    */
   href?: string;
+  /**
+   * Flag to indicate if the node is currently active.
+   */
+  isActive?: boolean;
+  /**
+   * Optional function to get the active state. This function is called whenever the location changes.
+   */
+  getIsActive?: (location: Location) => boolean;
```

Comment:
Does this override all other URL-based active paths? Or are URL-base paths still present in the active nodes array? Will this take priority by being first?